mxcube3/core/components/queue.py

```diff
@@ -2313,6 +2332,11 @@ def get_default_task_parameters(self, task_name):
 
         schema = self.get_task_schema(data_model) if data_model else {}
 
+        try:
```

Comment:
This looks like a system for controlling the UI rendering. Maybe I should adapt my GPhL UI to use this - but someone would have to explain it to me first.